bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -173,13 +173,23 @@
             .withDescription("Controls the color of the light").withCategory("ColorLight").build();
 
     /**
-     * Color-temperature: default system wide {@link ChannelType} which allows changing the color temperature
+     * Color-temperature: default system wide {@link ChannelType} which allows changing the color temperature in percent
      */
     public static final ChannelType SYSTEM_COLOR_TEMPERATURE = ChannelTypeBuilder
             .state(new ChannelTypeUID(BINDING_ID, "color-temperature"), "Color Temperature", "Dimmer")
-            .withDescription("Controls the color temperature of the light").withCategory("ColorLight")
-            .withStateDescriptionFragment(StateDescriptionFragmentBuilder.create().withMinimum(BigDecimal.ZERO)
-                    .withMaximum(new BigDecimal(100)).withPattern("%d").build())
+            .withDescription("Controls the color temperature of the light from 0 (cold) to 100 (warm)")
+            .withCategory("ColorLight").withStateDescriptionFragment(StateDescriptionFragmentBuilder.create()
+                    .withMinimum(BigDecimal.ZERO).withMaximum(new BigDecimal(100)).withPattern("%d").build())
```

Comment:
wouldn't it be safer to also use `%.0f` here, since PercentTypes are floating point numbers?